backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -507,39 +509,39 @@ func TestListExperiments_Failed(t *testing.T) {
 	clientManager.DB().Close()
 	_, err = server.ListExperiments(nil, &apiV2beta1.ListExperimentsRequest{})
 	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), "List experiments failed.")
-}
-
-func TestListExperimentsV1_SingleUser_NamespaceNotAllowed(t *testing.T) {
-	clientManager := resource.NewFakeClientManagerOrFatal(util.NewFakeTimeForEpoch())
-	resourceManager := resource.NewResourceManager(clientManager)
-	server := ExperimentServer{resourceManager: resourceManager, options: &ExperimentServerOptions{CollectMetrics: false}}
-	experiment := &apiV1beta1.Experiment{Name: "ex1", Description: "first experiment"}
-
-	_, err := server.CreateExperimentV1(nil, &apiV1beta1.CreateExperimentRequest{Experiment: experiment})
-	assert.Nil(t, err)
-	_, err = server.ListExperimentsV1(nil, &apiV1beta1.ListExperimentsRequest{
-		ResourceReferenceKey: &apiV1beta1.ResourceKey{
-			Type: apiV1beta1.ResourceType_NAMESPACE,
-			Id:   "ns1",
-		},
-	})
-	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), "In single-user mode, ListExperimentsV1 cannot filter by namespace.")
+	assert.Contains(t, err.Error(), "List experiments failed")
 }
 
-func TestListExperiments_SingleUser_NamespaceNotAllowed(t *testing.T) {
-	clientManager := resource.NewFakeClientManagerOrFatal(util.NewFakeTimeForEpoch())
-	resourceManager := resource.NewResourceManager(clientManager)
-	server := ExperimentServer{resourceManager: resourceManager, options: &ExperimentServerOptions{CollectMetrics: false}}
-	experiment := &apiV2beta1.Experiment{DisplayName: "ex1", Description: "first experiment"}
-
-	_, err := server.CreateExperiment(nil, &apiV2beta1.CreateExperimentRequest{Experiment: experiment})
-	assert.Nil(t, err)
-	_, err = server.ListExperiments(nil, &apiV2beta1.ListExperimentsRequest{Namespace: "ns1"})
-	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), "Invalid ListExperiments request. Namespace should not be provided in single-user mode.")
-}
+// func TestListExperimentsV1_SingleUser_NamespaceNotAllowed(t *testing.T) {
```

Comment:
Related to another comment above. Should we keep these tests? There's always a possibility the user may provided a namespace value in API requests. And we should be certain on our reaction and verify the behavior via tests.

Reply:
Repeating my response here: 
>In a single-user mode we simply replace the namespace with the default value (empty string). New namespaces will not be created. When creating k8s resources, we will use the POD_NAMESPACE from the config (Set to kubeflow). This logic will probably be more robust especially for some edge cases with people migrating from v1 to v2. No strong opinion here.

Is a user provides a namespace, we will replace it with the default value (empty string).